integration_test/gce/gce_testing.go

```diff
@@ -16,69 +16,64 @@
 
 /*
 Package gce holds various helpers for testing the agents on GCE.
+
 To run a test based on this library, you can either:
 
 * use Kokoro by triggering automated presubmits on your change, or
 * use "go test" directly, after performing the setup steps described
-in README.md.
+  in README.md.
 
 NOTE: When testing Windows VMs without using Kokoro, PROJECT needs to be
-a project whose firewall allows WinRM connections.
-[Kokoro can use stackdriver-test-143416, which does not allow WinRM
-connections, because our Kokoro workers are also running in that project.]
+    a project whose firewall allows WinRM connections.
+    [Kokoro can use stackdriver-test-143416, which does not allow WinRM
+    connections, because our Kokoro workers are also running in that project.]
 
 NOTE: This command does not actually build the Ops Agent. To test the latest
-
-Ops Agent code, first build and upload a package to Rapture. Then look up
-the REPO_SUFFIX for that build and add it as an environment variable to the
-command below; for example: REPO_SUFFIX=20210805-2. You can also use
-AGENT_PACKAGES_IN_GCS, for details see README.md.
-
-	PROJECT=dev_project \
-	ZONE=us-central1-b \
-	PLATFORMS=debian-10,centos-8,rhel-8-1-sap-ha,sles-15,ubuntu-2004-lts,windows-2012-r2,windows-2019 \
-	go test -v ops_agent_test.go \
-	  -test.parallel=1000 \
-	  -tags=integration_test \
-	  -timeout=4h
+    Ops Agent code, first build and upload a package to Rapture. Then look up
+    the REPO_SUFFIX for that build and add it as an environment variable to the
+    command below; for example: REPO_SUFFIX=20210805-2. You can also use
+	AGENT_PACKAGES_IN_GCS, for details see README.md.
+
+PROJECT=dev_project \
```

Comment:
Isn't this a command? It should be indented with 1 tab (https://tip.golang.org/doc/comment#code). We can do all lines with the same indent.

I thought we've addressed these formatting issues in #719?

Reply:
bad merge